doc/unbound.conf.5.in

```diff
@@ -490,20 +492,11 @@ This option defaults to 3000 milliseconds.
 Enable or disable EDNS TCP Keepalive. Default is no.
 .TP
 .B edns-tcp-keepalive-timeout: \fI<msec>\fR
-The period Unbound will wait for a query on a TCP connection when
-EDNS TCP Keepalive is active. If this timeout expires Unbound closes
-the connection. If the client supports the EDNS TCP Keepalive option,
+Overrides \fBtcp\-idle\-timeout\fR when EDNS TCP Keepalive is active.
```

Comment:
Perhaps "active" is a word commonly used in the Unbound documentation and its meaning is clear, but my concern here would be is it clear that "active" means "edns-tcp-keepalive = yes" rather than say "the current request included an edns-tcp-keepalive option or some other interpretation?

Reply:
@ximon18 fair comment, is the new text better?